docs/development.md

```diff
@@ -221,7 +221,7 @@ If you want to read probot's configuration from the same environment variables a
 const { createNodeMiddleware, createProbot } = require("probot");
 const app = require("./index.js");
 
-module.exports = createNodeMiddleware(app, { probot: createProbot() });
+module.exports = createNodeMiddleware(app, { probot: createProbot(), webhooksPath: '/path/to/webhook/endpoint' });
```

Comment:
The `webhooksPath` option is optional, your change implies that it might be required.

I'd suggest we add a new paragraph about customizing the route. It defaults to `/`

Reply:
done 👍

Reply:
wonderful, thanks a lot!